modules/nw-ingress-setting-update-subnet-loadbalancerservice.adoc

```diff
@@ -0,0 +1,83 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-update-subnet-Loadbalancerservice_{context}"]
+= Updating the subnets on an existing Ingress Controller
++
+You can update an `IngressController` with manually specified load balancer subnets in {product-title} to avoid any disruptions, to maintain the stability of your services, and to ensure that your network configuration aligns with your specific requirements. The following procedures show you how to select and apply new subnets, verify the configuration changes, and confirm successful load balancer provisioning. 
+
+.Procedure
+To update an `IngressController` with manually specified load balancer subnets, you can follow these steps:
+
+. Choose the subnets.
```

Comment:
I think it might be better to explain more detail here, that you are updating an existing ingresscontroller here (use your judgement in wording here):

```suggestion
. Modify the existing IngressController to update to the new subnets
```